testcase/integration/conftest.py

```diff
@@ -0,0 +1,268 @@
+
+    return account_addr
+
+
+def generate_channel_manage_data(channel_manage_data_path: str, config_path_list: list):
+    """Generate channel_manage_data.json
+
+    Args:
+        channel_manage_data_path (str): Desired path of channel_manage_data.json
+        config_path_list (list): Peer infos to be contained to the json file
+
+    """
+    channel_manage_data = {}
+    peers = []
+
+    for config_path in config_path_list:
+        with open(config_path) as f:
+            each_config = json.load(f)
+
+        for channel_name in each_config["CHANNEL_OPTION"]:
+            channel_manage_data[channel_name] = {
+                "score_package": "score/icx",
+            }
+
+        each_peer = {
+            "id": each_config["PEER_ID"],
+            "peer_target": f"[local_ip]:{each_config['PORT_PEER']}",
+            "order": each_config["PEER_ORDER"]
+        }
+        peers.append(each_peer)
+
+    for channel_name in channel_manage_data:
+        channel_manage_data[channel_name]["peers"] = peers
+
+    with open(channel_manage_data_path, "w") as f:
+        json.dump(channel_manage_data, f)
+
+
+def generate_genesis_file(genesis_path: str, accounts: list):
+    """
+    Generate genesis file
+
+    It mimics genesis block tx.
+    It will be contained to genesis block in this test.
+
+    Args:
+        genesis_path (str): Desired path of genesis file
+        accounts (list): This will be contained to genesis tx. Sample schema is below.
+            [
+                {
+                    "name": "treasury",
+                    "address": "hx1000000000000000000000000000000000000000",
+                    "balance": "0x1111"
+                },
+                ...
+            ]
+    """
+    genesis_data = {
+        "transaction_data": {
+            "nid": "0x3",
+            "accounts": accounts,
+            "message":
+                "A rHizomE has no beGInning Or enD; "
+                "it is alWays IN the miDDle, between tHings, interbeing, intermeZzO. ThE tree is fiLiatioN, "
+                "but the rhizome is alliance, uniquelY alliance. "
+                "The tree imposes the verb \"to be\" but the fabric of the rhizome is the conJUNction, "
+                "\"AnD ... and ...and...\""
+                "THis conJunction carriEs enouGh force to shaKe and uproot the verb \"to be.\" "
+                "Where are You goIng? Where are you coMing from? What are you heading for? "
+                "These are totally useless questions.\n\n- 'Mille Plateaux', Gilles Deleuze & Felix Guattari\n\n\""
+                "Hyperconnect the world\""
+        }
+    }
+
+    with open(genesis_path, "w") as f:
+        json.dump(genesis_data, f)
+
+
+def get_peer_info(conf_path_list: list, order: int = 0):
+    """Get peer information"""
+    target_peer_conf_path = conf_path_list[order]
+
+    with open(target_peer_conf_path) as f:
+        default_peer_conf = json.load(f)
+
+    return default_peer_conf
```

Comment:
I think it's confusing to return `default_peer_conf` when method_name is generally getting peer_info. If its usage is just for the default peer info, the method name should be changed instead.

Reply:
I agree that variable names used in those line raise terrible confusion. So I changed.

I intended that the function `get_peer_info` helps extracting `order`'s peer information in given list. It means, the expected return value is should be peer info, as dict type.

So, I changed confusing variable `default_peer_conf` to simply `peer_info`